src/Uno.UI/Microsoft/UI/Xaml/Controls/NumberBox/MathToken.cs

```diff
@@ -1,4 +1,5 @@
-﻿namespace Microsoft.UI.Xaml.Controls
+﻿// MUX Reference NumberBoxParser.h, commit de788345659ba319597161149843504fbe686659
```

Comment:
Please add Microsoft copyright headers to files ported from WinUI (they should be on top of most MUX classes we have in the repo, so you can copy from there)

Reply:
```suggestion
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See LICENSE in the project root for license information.
﻿// MUX Reference NumberBoxParser.h, commit de788345659ba319597161149843504fbe686659
```